exporter/metrics.go

```diff
@@ -74,12 +102,18 @@ func makeRawMetric(name string, value interface{}, labels map[string]string) (pr
 		return nil, fmt.Errorf("makeRawMetric: unhandled type %T", v)
 	}
 
-	fqName := fqMetricName(name)
+	if labels == nil {
+		labels = make(map[string]string)
+	}
+
+	fqName := prometheusize(prefix + name)
+	if label, ok := nodeToPDMetrics[prefix]; ok {
+		fqName = prometheusize(prefix)
+		labels[label] = name
```

Comment:
This code looks weird.

Reply:
This is because we have "hardcoded" metric and label names as defined in the development proposal. https://confluence.percona.com/display/~akira.kurogane/2020+MongoDB+Prometheus+Exporter+-+User+Story
So, we take the name we got following the tree structure and then, if than name matches a predefined name/label pair, we must use the predefined one.